docs/www/guide-wasm-elastic.md

```diff
@@ -0,0 +1,218 @@
+# Redpanda wasm function + Elastic Search
```

Comment:
@andresaristizabal Looks like a great article!
Looks like we need to correct the style on these terms:

- wasm -> Wasm (as shown in https://webassembly.org/)
- ElasticSearch and Elastic Search -> Elasticsearch (as shown in https://www.elastic.co/elasticsearch/)

Do you want me to add that as a suggestion for every instance?

Reply:
Got it, let me rename all words on this guide.